ERROR - 2025-02-27 14:34:42 --> mysqli_sql_exception: Can't create table `sistem_ujian`.`users` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\SistemUjian\app\Database\Migrations\2025-02-27-143039_SitemUjian.php(24): CodeIgniter\Database\Forge->createTable('users')
#5 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(866): App\Database\Migrations\SitemUjian->up()
#6 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\SistemUjian\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-02-27 14:42:41 --> mysqli_sql_exception: Unknown table 'sistem_ujian.log_aktivitas' in C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `log...', 0)
#1 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `log...')
#2 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `log...')
#3 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `log...')
#4 C:\xampp\htdocs\SistemUjian\app\Database\Migrations\2025-02-27-143039_SitemUjian.php(106): CodeIgniter\Database\Forge->dropTable('log_aktivitas')
#5 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(866): App\Database\Migrations\SitemUjian->down()
#6 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\SistemUjian\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-02-27 14:43:02 --> mysqli_sql_exception: Unknown table 'sistem_ujian.log_aktivitas' in C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `log...', 0)
#1 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `log...')
#2 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `log...')
#3 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `log...')
#4 C:\xampp\htdocs\SistemUjian\app\Database\Migrations\2025-02-27-143039_SitemUjian.php(106): CodeIgniter\Database\Forge->dropTable('log_aktivitas')
#5 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(866): App\Database\Migrations\SitemUjian->down()
#6 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\SistemUjian\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
CRITICAL - 2025-02-27 14:55:15 --> ErrorException: Cannot declare class App\Database\Seeds\KelasSeeder, because the name is already in use
[Method: CLI, Route: db:seed/DatabaseSeeder]
in APPPATH\Database\Seeds\DatabaseSeeder.php on line 7.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-27 15:03:22 --> ErrorException: Cannot declare class App\Database\Seeds\MapelSeeder\MapelSeeder, because the name is already in use
[Method: CLI, Route: db:seed/DatabaseSeeder]
in APPPATH\Database\Seeds\MapelSeeder.php on line 7.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2025-02-27 15:04:19 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sistem_ujian`.`soal`, CONSTRAINT `soal_ujian_id_foreign` FOREIGN KEY (`ujian_id`) REFERENCES `ujian` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `so...', 0)
#1 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', NULL, false)
#4 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\UjianSeeder.php(29): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\UjianSeeder->run()
#7 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\DatabaseSeeder.php(14): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\DatabaseSeeder->run()
#9 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\SistemUjian\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2025-02-27 15:05:30 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sistem_ujian`.`soal`, CONSTRAINT `soal_ujian_id_foreign` FOREIGN KEY (`ujian_id`) REFERENCES `ujian` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `so...', 0)
#1 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', NULL, false)
#4 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\UjianSeeder.php(29): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\UjianSeeder->run()
#7 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\DatabaseSeeder.php(14): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\DatabaseSeeder->run()
#9 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\SistemUjian\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2025-02-27 15:07:56 --> mysqli_sql_exception: Unknown column 'nama' in 'field list' in C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `uj...', 0)
#1 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `uj...')
#2 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `uj...')
#3 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `uj...', NULL, false)
#4 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\UjianSeeder.php(29): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\UjianSeeder->run()
#7 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\DatabaseSeeder.php(14): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\DatabaseSeeder->run()
#9 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\SistemUjian\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
ERROR - 2025-02-27 15:10:13 --> mysqli_sql_exception: Unknown column 'waktu' in 'field list' in C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `uj...', 0)
#1 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `uj...')
#2 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `uj...')
#3 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `uj...', NULL, false)
#4 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\UjianSeeder.php(29): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\UjianSeeder->run()
#7 C:\xampp\htdocs\SistemUjian\app\Database\Seeds\DatabaseSeeder.php(14): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Database\Seeder.php(147): App\Database\Seeds\DatabaseSeeder->run()
#9 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#10 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#11 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#12 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#13 C:\xampp\htdocs\SistemUjian\vendor\codeigniter4\framework\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#14 C:\xampp\htdocs\SistemUjian\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#15 {main}
